src/routes/(auth)/login/+server.ts

```diff
@@ -27,9 +27,9 @@ export async function createToken(
 
   if (response?.error || response?.data?.tokenCreate?.error) {
     if (response?.data?.tokenCreate?.error) {
-      const error = response?.data?.tokenCreate?.error;
+      const error = response.data.tokenCreate.error;
```

Comment:
This line here is a `UserError` instance, I think you can take advantage of that.

```suggestion
      const error: UserError = response.data.tokenCreate.error;
```

This will allow you to have type definitions for the code field.